babel.config.js

```diff
@@ -5,8 +5,15 @@ module.exports = function (api) {
       strictMode: true,
     },
     targets: {
-      browsers: ['chrome >= 80', 'firefox >= 78'],
+      browsers: ['chrome >= 89', 'firefox >= 89'],
     },
+    plugins: [
+      // `browserify` is old and busted, and doesn't support `??=` (and other
+      // logical assignment operators). This plugin lets us target es2020-level
+      // browsers (except we do still end up with transpiled logical assignment
+      // operators 😭)
+      '@babel/plugin-transform-logical-assignment-operators',
```

Comment:
We'll need to list this in the depcheck config, as depcheck doesn't see where it's used